opencti-platform/opencti-front/src/private/components/nav/LeftBar.jsx

```diff
@@ -715,48 +715,50 @@ const LeftBar = () => {
             )}
           </MenuList>
         </Security>
-        <Security needs={[EXPLORE]}>
+        <Security needs={[EXPLORE, MODULES, KNOWLEDGE, TAXIIAPI_SETCOLLECTIONS, TAXIIAPI_SETCSVMAPPERS]}>
```

Comment:
Here we had 2 <Security> on inside the other, and the top one was not including "MODULES, KNOWLEDGE, TAXIIAPI_SETCOLLECTIONS" that was used inside.

So it was like:
```
<Security needs={[EXPLORE]}>
    <Security needs={[MODULES, KNOWLEDGE, TAXIIAPI_SETCOLLECTIONS]}>
    </Security>
</Security>
```
Now it's:
```
<Security needs={[EXPLORE, MODULES, KNOWLEDGE, TAXIIAPI_SETCOLLECTIONS, TAXIIAPI_SETCSVMAPPERS]}>
    <Security needs={[EXPLORE]}>
    </Security>
    <Security needs={[EMODULES, KNOWLEDGE, TAXIIAPI_SETCOLLECTIONS, TAXIIAPI_SETCSVMAPPERS]}>
    </Security>
</Security>
```